frontend/src/v5/ui/routes/dashboard/projects/federations/federationsList/federationListItem/federationListItem.component.tsx

```diff
@@ -78,11 +71,9 @@ export const FederationListItem = memo(({
 		}
 	};
 
-	const onClickEdit = () => {
-		DialogsActionsDispatchers.open(EditFederationModal, {
-			federation,
-		});
-	};
+	// eslint-disable-next-line max-len
+	const onClickSettings = () => DialogsActionsDispatchers.open(FederationSettingsForm, { federationId: federation._id });
+	const onClickEdit = () => DialogsActionsDispatchers.open(EditFederationModal, { federation });
```

Comment:
We are using _FederationSettings**Form**_ and _EditFederation**Modal**_. I know they were already named this way, but could you please rename them so that they use a common suffix?